src/indicators/info.js

```diff
@@ -0,0 +1,94 @@
+import { useState } from 'react'
+import uniqid from 'uniqid'
+import time from '../destructurers/time'
+
+const getStopInfo = (stopData) => {
+  const stops = []
+  let timestamp = 'unknown'
+
+  Object.values(stopData).forEach((stop) => {
```

Comment:
tiny semantic note. you're using 'stop' inconsistently here for both a source of data, and something that gets pushed into an array of stops.

if you renamed `stop` on line 9 to `dataPoint` and then destructured just the values you needed into a stop:Stop object that could push into the stops array, after you add in Typescript of course, you would have a really clear flow of data to data structure.

Reply:
Ugh. Yes. And in Tracker.js there's `stops` and `busStops`. Totally ambiguous and been chafing me for days.

I seriously need to get experience with TypeScript, but maybe a little late for this project. Next one. But I now I deffo need to work on meaningful variable names. I like your `dataPoint`, and maybe `thingSettings` when I load data from `settings/thing`...